Dockerfile

```diff
@@ -13,28 +13,18 @@ RUN go get github.com/tools/godep
 RUN echo "Installing the go-bindata tool"
 RUN go get github.com/jteeuwen/go-bindata/...
 
-ADD . /src/github.com/
+ADD . /src/github.com/openwhisk/openwhisk-cli
 
 # Load all of the dependencies from the previously generated/saved godep generated godeps.json file
 RUN echo "Restoring Go dependencies"
-RUN cd /src/github.com/go-whisk-cli && /bin/godep restore -v
+RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/godep restore -v
 
-# Collect all translated strings into single .go module
-RUN echo "Generating i18n Go module"
-RUN cd /src/github.com/go-whisk-cli && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
```

Comment:
@pritidesai, in OpenWhisk, lines 24 and 25 are used to take the externalized strings (https://github.com/openwhisk/openwhisk-cli/blob/master/wski18n/resources/en_US.all.json) and incorporate them into the generated Wsk CLI binaries. Is this being done elsewhere?